### PR TITLE
\hyperlink{execenv}{execution environment} and linkdest

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -723,7 +723,7 @@ \section{Contract Creation}\label{ch:create}\hypertarget{endow}{}
 I_{\mathrm{s}} & \equiv & s \\
 I_{\mathrm{v}} & \equiv & v \\
 I_{\mathbf{b}} & \equiv & \mathbf{i} \\
-I_{\mathrm{e}} & \equiv & e \\
+\hyperlink{I__v}{I_{\mathrm{v}}} & \equiv & e \\
```
Comment (pirapira) [inline Paper.tex]: Why did I_e turn into I_v?

Comment (jamesray1) [reply Paper.tex]: Wrong line.

**Comment by jamesray1** [comment]
@pirapira, I'm not sure if you got a notification of my commit but this should be OK to merge now. Please also have a look at other PRs, including yours truly, as there are several that are open.

**Comment by pirapira** [comment]
I'm clearing my mailbox.  I arrived here now.

File: Paper.tex [hunk 2]
```diff
@@ -877,21 +877,21 @@ \subsection{Fees Overview}
...
-\item\linkdest{exec_call_or_create_depth_I__e}{} $I_{\mathrm{e}}$, the depth of the present message-call or contract-creation (i.e. the number of {\small CALL}s or {\small CREATE}s being executed at present).
-\item\linkdest{exec_permission_to_modify_state_I__w}{} $I_{\mathrm{w}}$, the permission to make modifications to the state.
+\item\linkdest{Wei_value_exec_I__v}{}\linkdest{I__v}{} $I_{\mathrm{v}}$, the value, in Wei, passed to this account as part of the same procedure as execution; if the execution agent is a transaction, this would be the transaction value.
+\item\linkdest{exec_machine_code_I__b}{}\linkdest{I__b} $I_{\mathbf{b}}$, the byte array that is the machine code to be executed.
+\item\linkdest{exec_block_header_I__H}{}\linkdest{I__H} $I_{\mathbf{H}}$, the block header of the present block.
+\item\linkdest{exec_call_or_create_depth_I__e}{}\linkdest{I__e} $I_{\mathrm{e}}$, the depth of the present message-call or contract-creation (i.e. the number of {\small CALL}s or {\small CREATE}s being executed at present).
+\item\linkdest{exec_permission_to_modify_state_I__w}{}\linkdest{I__e} $I_{\mathrm{w}}$, the permission to make modifications to the state.
```
Comment (pirapira) [inline Paper.tex]: I think it should be `I__w` not `I__e`.

Comment (jamesray1) [reply Paper.tex]: Yep!

**Pull request merged**